config.php

```diff
@@ -3,7 +3,8 @@
 #General Informations
 $CONFIG_CONTACT = 'fsi@fsi.uni-tuebingen.de';
 $CONFIG_TERM = 'WS 20/21'; # example: WS 19/20 or SS 20
-$fp = "../../eei-registration/"; #File Prefix
+// see https://www.php.net/manual/en/language.constants.predefined.php
+$fp = "__DIR__/../eei-registration/"; #File Prefix
```

Comment:
I think this needs to be:
```suggestion
$fp = __DIR__ . "/../eei-registration/"; #File Prefix
```

(my example from the last review was bad)